docs/book/compatiblity_matrix.md

```diff
@@ -21,3 +21,4 @@ Note:
 
 - vSphere CSI driver is not supported on Windows based vCenter.
 - vSphere CSI driver is not supported on vSAN stretch cluster.
+- vSphere CSI driver does not support provisioning block or raw block volumes on the NFSv4 Datastore.
```

Comment:
How about this - `vSphere CSI driver does not support provisioning block or raw block volumes on the NFSv4 Datastore in the vCenter server.`

Reply:
Let us alos point to the FCD public documentation on this constraint as the natural question from the customer would be why is it not supported.

Reply:
I don't think we have any public FCD documentation available which says about this constraint. If you come across let me know I will tag it here.

Reply:
If we do not have any public facing doc, we need to mention why we do not support NFSv4 datastore in CNS. Can you find out the reasoning from the dependent team and put a summary here?

Reply:
"vSphere CSI Driver uses CNS API. CNS uses FCD and FCD depends on rename atomicity provided by POSIX filesystems. Currently, there is an open bug that needs a fix in the NFSv4 client where this atomicity is broken. So, FCD and NFSv4 is not supported in NFSv4 till the bug is fixed."